xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/objects/classes/DBListClass.java

```diff
@@ -594,4 +609,15 @@ public void displayView(StringBuffer buffer, String name, String prefix, BaseCol
             buffer.append(getDisplayValue(prop.getValue(), name, map, context));
         }
     }
+
+    private String getPropertyType(String className, String propertyName, XWikiContext context) throws Exception
+    {
+        PropertyClass pc = null;
+        try {
+            pc = (PropertyClass) context.getWiki().getDocument(className, context).getXClass().get(propertyName);
+        } catch (XWikiException e) {
+            LOGGER.error("Failed to get property [{}] for document [{}]", propertyName, className, e);
+        }
+        return pc.newProperty().getClass().getName();
```

Comment:
Hello Marius ,
I've add a default value to null on the getQuery() function , as you can see it on my last commit , here :
https://github.com/zoubir/xwiki-platform/commit/5fffe0a254b14ad193ef1a969624ebb7e56d721b#diff-712bf30d51cacf05af1e8ef2c8898181R202

Thanks